governance/default_test.go

```diff
@@ -537,3 +537,24 @@ func TestCypressGenesisHash(t *testing.T) {
 		t.Errorf("Generated hash is not equal to Cypress's hash. Want %v, Have %v", cypressHash.String(), block.Hash().String())
 	}
 }
+
+func TestWriteGovernance_idxCache(t *testing.T) {
+	gov := getGovernance()
+
+	tstMap := copyMap(testGovernanceMap)
+
+	src := NewGovernanceSet()
+	delta := NewGovernanceSet()
+	src.Import(tstMap)
+
+	gov.WriteGovernance(30, src, delta)
```

Comment:
if return err checking is added, it is better. :)

Reply:
@KimKyungup Thanks. I changed the code to check an error in calling WriteGovernance.